### PR TITLE
Support SizeLongConvertor for K, M, G, T suffix sizes

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/wire/converter/SizeLongConverter.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+     * @param text The text to parse, containing a numeric value followed by
+     *             a size suffix (K, M, G, T). Case-insensitive.
+     * @return The parsed long value, adjusted for the size suffix. If no
+     * suffix is provided, returns the plain numeric value.
+     * @throws NumberFormatException If the text before the size suffix is not
+     *                               a valid long number or if the text is empty.
+     */
+    @Override
+    public long parse(CharSequence text) throws NumberFormatException {
+        int shift = 0;
```
Comment (tgd) [inline src/main/java/net/openhft/chronicle/wire/converter/SizeLongConverter.java]: Can consider throwing NPE with custom message if `text==null`

**Review by tgd — APPROVED** [review]

**Comment by tgd** [comment]
Nice one @peter-lawrey - be great to have this in the REAMDE also

**Review by JerryShea — APPROVED** [review]
cool

File: src/main/java/net/openhft/chronicle/wire/converter/SizeLongConverter.java (hunk 1 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/wire/converter/SizeLongConverter.java]: it will on the next line.

**Comment by peter-lawrey** [comment]
@tgd Added https://github.com/OpenHFT/Chronicle-Wire/issues/861 for adding README

**Pull request merged**